docs/specs/source_python_math.tex

```diff
@@ -5,9 +5,9 @@ \subsection*{MATH Library}
 \begin{itemize}
 \item \href{https://sourceacademy.org/sicpjs/1.1.4\#p8}{\lstinline{math_}$\textit{name}$},
 where $\textit{name}$ is any name specified in the
-JavaScript
+Python
 \texttt{Math} library, see\\
-\href{https://www.ecma-international.org/ecma-262/9.0/index.html\#sec-math-object}{\color{DarkBlue}ECMAScript Specification, Section 20.2}. Examples:
+\href{https://docs.python.org/3.7/library/math.html}{\color{DarkBlue}Python 3.7.17 Documentation}. Examples:
```

Comment:
Don't you want to go for the latest stable release? https://docs.python.org/3/library/math.html

Are you implementing all functions in the Python math library? How about math.isnan, or math.factorial. If any functions are missing, please create an issue where you list what needs to be done to make the implementation consistent with the specs.